wls-gui-wahllokalsystem/src/components/ergebnisermittlung/MBW/stapelAB/TheMBWGueltigeStimmenAnzeigenTable.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container>
+    <v-table striped="odd">
```

Comment:
bisher wir in Tabelle kein Striped. Das Designthema ist mit #1630 auf dem Schirm. Darin ist auch die Frage nach Stribed enthalten

```suggestion
    <v-table>
```

Reply:
resolved with 78f6d6a48a97017fbd89bad07e9ccf2d68868ef8